### PR TITLE
ghIssues.pl - fix Cheat Sheets tab name and edit permissions

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
ghIssues used to assign the "Answer" tab to all goodies, but for Cheat Sheets it should be "Cheat Sheet" instead.
Also fixed giving edit permissions to developers.

### CODE REVIEW

File: script/ghIssues.pl [hunk 1]
```diff
@@ -561,8 +564,11 @@ sub add_developer {
     try{
         if($user){
             my $ddgc_name = $user->username;
+            
             # Give edit permissions to the contributor
-            $ia->add_to_users($user);
+            my $ia = $d->rs('InstantAnswer')->find({id => $ia_hash->{id}});
+            $ia->add_to_users($user) unless ($ia->users->find($user->id) || $user->admin);
```
Comment (jdorweiler) [inline script/ghIssues.pl]: Can you add a log to the catch block too?  I forgot to do that.

Comment (MariagraziaAlastra) [inline script/ghIssues.pl]: @jdorweiler Done, please let me know if it's ok :)

Comment (jdorweiler) [inline script/ghIssues.pl]: Cool thanks,  Can you check that `$ia` exists here before using it?  Now I'm just trying to be cautious.

**Comment by MariagraziaAlastra** [comment]
@jdorweiler it should be safer now :smile:

**Pull request merged**